consensus/service/src/tx_manager.rs

```diff
@@ -129,7 +131,7 @@ pub struct TxManager<E: ConsensusEnclaveProxy, UI: UntrustedInterfaces> {
     cache: Arc<Mutex<HashMap<TxHash, CacheEntry>>>,
```

Comment:
Does this still needs an Arc?

Reply:
Good catch. It probably does not need that Arc. Will try.